duckduckgo/Android
# Translations for updated About screen

Task/Issue URL: https://app.asana.com/0/0/1205786903855013/f

### Description
Translations for new About screen content

### Steps to test this PR

- Change device language
- Install from branch
- Go to Settings>About screen
- Check screen is translated

### No UI changes
